src/hooks/core/chat/useSystemMessage.ts

```diff
@@ -120,16 +123,19 @@ const useSystemMessage = () => {
     while (sortedMessages?.length > 0) {
       const checkFirstMessage = sortedMessages[0];
       if (!__isSystemMessage(checkFirstMessage)) {
+        if (withLog) console.log('checkpoint1', checkFirstMessage);
         firstMessage = checkFirstMessage;
         break;
       }
 
       if (checkFirstMessage?.text?.toLocaleLowerCase()?.includes('this topic has new')) {
+        if (withLog) console.log('checkpoint2');
         sortedMessages?.shift();
         continue;
       }
 
       if (checkFirstMessage?.text?.toLocaleLowerCase()?.includes('you joined this community')) {
+        if (withLog) console.log('checkpoint3');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `console.log` for debugging purposes is not a good practice for production code. It's better to use a logging library that can be toggled on/off based on the environment (development/production). If this is only for development, consider removing these before merging to the main branch.



<!-- This is an auto-generated comment by OSS CodeRabbit -->